app/tests/v1/test_question.py

```diff
@@ -1,39 +1,32 @@
 '''This module represents tests for the question entity'''
 import json
 
+from app.api.v1.models.meetup_model import MEETUPS
 from app.tests.v1.test_base import BaseTestCase
 
 class QuestionTestCase(BaseTestCase):
     '''Test definitions for a question'''
     def test_create_question(self):
         '''Test the API can create a question'''
-        res = self.get_response_from_user_login(self.admin_registration, self.admin_login)
-        response_msg = json.loads(res.data.decode("UTF-8"))
-        access_token = response_msg["access_token"]
-        res = self.client().post(
-            '/api/v1/meetups',
-            headers=self.get_authentication_headers(access_token),
-            data=json.dumps(self.meetup)
-        )
-        self.assertEqual(res.status_code, 201)
-        res = self.get_response_from_user_login(self.user_registration, self.user_login)
-        response_msg = json.loads(res.data.decode("UTF-8"))
-        access_token = response_msg["access_token"]
+        access_token = self.get_access_token(self.admin_registration, self.admin_login)
+        self.create_meetup(access_token, self.meetup)
+        access_token = self.get_access_token(self.user_registration, self.user_login)
         res = self.client().post(
             '/api/v1/meetups/1/questions',
             headers=self.get_authentication_headers(access_token),
             data=json.dumps(self.question)
         )
+        response_msg = json.loads(res.data.decode("UTF-8"))
         self.assertEqual(res.status_code, 201)
+        self.assertTrue(response_msg['data'])
+        self.assertEqual(
+            MEETUPS[0]['questions'][0]['question_id'],
+            response_msg['data'][0]['questions'][0]['question_id']
+        )
 
```

Comment:
Good job asserting the contents of the data returned as well.

Reply:
Thank you.